test_api.py

```diff
@@ -1,19 +1,28 @@
 from api_helper import ShoonyaApiPy
 import logging
+from dotenv import load_dotenv
  
 #enable dbug to see request and responses
 logging.basicConfig(level=logging.DEBUG)
 
 #start of our program
 api = ShoonyaApiPy()
 
-#credentials
-user    = <uid>
-pwd     = <password>
-factor2 = <2nd factor>
-vc      = <vendor code>
-app_key = <secret key>
+#create .env file to securely access user credentials in program
+load_dotenv('/home/akash/Documents/Projects/.env')
+
+#totp from shoonya dashboard security totp section
```

Comment:
Not clear what this is referring to.

Is this suggesting to fetch the value of `totp` from some URL?
If yes, maybe mention the exact details here. including a format/template of what a value would looks like.